docs/section18/Rule_18-1

```diff
@@ -0,0 +1,72 @@
+# HVAC_SystemZoneAssignment - Rule 18-1  
+**Schema Version:** 0.0.22  
+
+**Rule ID:** 18-1  
+**Rule Description:** Buildings in CZ 0-3A with predominant HVAC BAT = public assembly, <120,000 ft2 have baseline HVAC system type 4 (PSZ HP)
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 18 HVAC_SystemZoneAssignment  
+**Appendix G Section Reference:** Table G3.1.1.3
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. Buildings in CZ 0-3A
+2. predominant HVAC BAT = public assembly
+3. building area < 120,0000 ft2
+4. the zone does not meet one if the G3.1.1.1 exceptions
+
+**Manual Check:** Yes  
+**Evaluation Context:** Evaluate whether each Zone is served by the correct HVAC system type  
+**Data Lookup:**   
+**Function Call:** 
+
+1. zone_meets_any_G3_1_1_exception(P-RMR,B-RMR,zone)
```

Comment:
We don't yet have these function, right? Perhaps it makes sense to develop it before using it, so that we know the logic and returned variables. You defined function [zone_meets_G3_1_1c](https://github.com/pnnl/ruleset-checking-tool/pull/683), but this particular exception does not apply to System 4 that is checked below.